examples/deepspeed_mii/DeepSpeed_mii_handler.py

```diff
@@ -0,0 +1,118 @@
+import json
+import logging
+import os
+import zipfile
+from abc import ABC
+
+import mii
+import numpy as np
+import torch
+
+from ts.torch_handler.base_handler import BaseHandler
+
+logger = logging.getLogger(__name__)
+logger.info("DeepSpeed MII version %s", mii.__version__)
+
+
+class DeepSpeedMIIHandler(BaseHandler, ABC):
+    """
+    Diffusers handler class for text to image generation.
+    """
+
+    def __init__(self):
+        self.initialized = False
+
+    def initialize(self, ctx):
+        """In this initialize function, the Stable Diffusion model is loaded and
+        initialized here.
+        Args:
+            ctx (context): It is a JSON Object containing information
+            pertaining to the model artefacts parameters.
+        """
+        self.manifest = ctx.manifest
+        properties = ctx.system_properties
+        model_dir = properties.get("model_dir")
+
+        self.device = torch.device(
+            "cuda:" + str(properties.get("gpu_id"))
+            if torch.cuda.is_available() and properties.get("gpu_id") is not None
+            else "cpu"
+        )
+        with zipfile.ZipFile(model_dir + "/model.zip", "r") as zip_ref:
```

Comment:
Thanks @msaroufim The stable-diffusion model has multiple components (https://huggingface.co/CompVis/stable-diffusion-v1-4/tree/main) and it will be difficult to include them in `--extra-files`. So i created a zip file and included in extra files.